internal/vendor/wiz.go

```diff
@@ -177,15 +177,15 @@ type UpdateSAMLIdentityProviderInput struct {
 
 // UpdateSAMLIdentityProviderPatch struct -- updates
 type UpdateSAMLIdentityProviderPatch struct {
-	EntityID                 string                        `json:"entityID,omitempty"`
-	LoginURL                 string                        `json:"loginURL,omitempty"`
-	LogoutURL                string                        `json:"logoutURL,omitempty"`
-	UseProviderManagedRoles  *bool                         `json:"useProviderManagedRoles,omitempty"`
-	AllowManualRoleOverride  *bool                         `json:"allowManualRoleOverride,omitempty"`
-	Certificate              string                        `json:"certificate,omitempty"`
-	Domains                  []string                      `json:"domains,omitempty"`
-	GroupMapping             []SAMLGroupMappingUpdateInput `json:"groupMapping,omitempty"`
-	MergeGroupsMappingByRole *bool                         `json:"mergeGroupsMappingByRole,omitempty"`
+	LoginURL                 string                        `json:"loginURL"`
+	LogoutURL                string                        `json:"logoutURL"`
+	IssuerURL                string                        `json:"issuerURL"`
+	UseProviderManagedRoles  *bool                         `json:"useProviderManagedRoles"`
+	AllowManualRoleOverride  *bool                         `json:"allowManualRoleOverride"`
+	Certificate              string                        `json:"certificate"`
+	Domains                  []string                      `json:"domains"`
+	GroupMapping             []SAMLGroupMappingUpdateInput `json:"groupMapping"`
+	MergeGroupsMappingByRole *bool                         `json:"mergeGroupsMappingByRole"`
```

Comment:
The omitempty tag is based on the GraphQL schema.  The golang struct should mirror the GraphQL schema where possible.  For this struct, all elements are optional.

```
input UpdateSAMLIdentityProviderPatch {
  name: String
  issuerURL: String
  loginURL: String
  logoutURL: String
  useProviderManagedRoles: Boolean
  allowManualRoleOverride: Boolean
  certificate: String
  domains: [String!]
  groupMapping: [SAMLGroupMappingUpdateInput!]
  mergeGroupsMappingByRole: Boolean
}
```